src/components/rubric-levels-mobile.js

```diff
@@ -574,47 +575,52 @@ class RubricLevelsMobile extends composeMixins(
 	}
 
 	_handleDrag(e) {
-		if (!this._isMousePressed || this.readOnly) {
+		if (!this._dragContext || this.readOnly) {
 			return;
 		}
-		// when mouse moves we know the drag has been initiated
-		this._currentDragContext.elementHasBeenDragged = true;
-		const scrollOffset = this._scroll - this._currentDragContext.origin.scroll;
-		const fromEdge = e.pageX - this._currentDragContext.bounds.left + scrollOffset;
-		const clampedLevel = Math.floor(fromEdge / this._currentDragContext.stepWidth);
+		const dx = Math.abs(this._dragContext.origin.x - e.pageX);
+		const dy = Math.abs(this._dragContext.origin.y - e.pageY);
+		if ((dx > DRAG_DISTANCE || dy > DRAG_DISTANCE) && !this._dragContext.isDragEvent) {
+			this._dragContext.isDragEvent = true;
+		} else if (dx <= DRAG_DISTANCE && dy <= DRAG_DISTANCE && !this._dragContext.isDragEvent) {
+			return;
+		}
```

Comment:
You can simplify this
```suggestion
if (!this._dragContext.isDragEvent) {
		if (dx > DRAG_DISTANCE || dy > DRAG_DISTANCE) {
			this._dragContext.isDragEvent = true;
		} else {
			return;
		}
	}
}
```

Reply:
Above line 589 is an extra bracket

Reply:
> Above line 589 is an extra bracket

Good catch, updated it